### PR TITLE
change public key to JsonWebKey

### PR DESCRIPTION
**Pull request by ququzone**
# Description
change public key to JsonWebKey

Fixes #(issue)

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Code refactor or improvement
- [] Breaking change (fix or feature that would cause a new or changed behavior of existing functionality)
- [] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] make test
- [] fullsync
- [] Other test (please specify)

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ioctl/cmd/did/diddoc.go [hunk 1]
```diff
@@ -146,18 +161,25 @@ func NewDIDDoc(publicKey []byte) (*Doc, error) {
 	doc := &Doc{
 		Context: []string{
 			KnownDIDContext,
+			JWSDIDContext,
 			Secp256k1DIDContext,
```
Comment (dustinxie) [inline ioctl/cmd/did/diddoc.go:165]: is this `Secp256k1DIDContext` still needed here?

Comment (dustinxie) [reply ioctl/cmd/did/diddoc.go:165]: if not, can delete its definition as well

File: ioctl/cmd/did/diddoc.go [hunk 2]
```diff
@@ -93,20 +105,24 @@ func (doc *Doc) AddService(tag, serviceType, endpoint string) {
...
+			doc.Service[i].ServiceEndpoint = endpoint
 			return
 		}
 	}
 	doc.Service = append(doc.Service, serviceStruct{
 		ID:              id,
 		Type:            serviceType,
 		ServiceEndpoint: endpoint,
+		RecipientKeys:   doc.Authentication,
+		Accept:          []string{"didcomm/v2"},
```
Comment (dustinxie) [inline ioctl/cmd/did/diddoc.go:125]: should we define this as a const, like other const variables?

**Review by dustinxie — APPROVED** [review]

**Comment by raullenchai** [comment]
Close b/c it has been inactive for 1+yr.

**Pull request closed without merging**